client/app/components/routes/select-operator/form/UserOperator.tsx

```diff
@@ -53,38 +58,47 @@ const createUserOperatorSchema = (
         .pc_business_structure,
       anyOf: businessStructureOptions,
     };
+
   return localSchema;
 };
 
 export default async function UserOperator({
   params,
 }: Readonly<{
-  params?: { id?: number };
+  params?: { id?: number; readonly?: boolean };
 }>) {
-  const serverError = <div>Server Error. Please try again later.</div>;
-
   const businessStructures: BusinessStructure[] | { error: string } =
     await getBusinessStructures();
 
   const userData: UserInformationInitialFormData | { error: string } =
     await getCurrentUser();
 
-  if ("error" in userData || "error" in businessStructures) return serverError;
+  // TODO: define schema of data returned from endpoint
+  const userOperatorData: any | { error: string } =
```

Comment:
would be ideal to replace this `any` with a new schema for the data structure being returned

Reply:
Fixed!